R/variantSetTests.R

```diff
@@ -167,7 +168,20 @@ testVariantSet <- function( nullmod, G, weights,
         err <- 1
         smmat.pval <- burden.pval
     }
-    return(list(pval_burden = burden.pval, pval_theta = theta.pval, pval_SMMAT = smmat.pval, err = err, pval_theta.method = out$pval.method))
+    return(list(
+      # burden test output.
+      Score_burden = U.sum,
+      Score.SE_burden = sqrt(V.sum),
+      Stat_burden = burden.stat,
```

Comment:
@amstilp -- for consistency with the regular burden test output, the Stat should probably be the sqrt of what you have here. i.e. U.sum/sqrt(V.sum). I also can't really decide if we even need to include the stat here. The burden test does return Score.Stat, but it's so easy to get from the Score and the SE...

Reply:
Fixed by taking the sqrt of burden.stat. Might as well return it?

Reply:
Actually, we should probably compute burden.stat as U.sum/sqrt(V.sum) and then square when calculating the p-value. The way it is now, we lose the sign of the test statistic. Sorry, didn't think of this til right now.

Reply:
oh! good point. fixed.